store/metric.go

```diff
@@ -75,7 +75,7 @@ func (m *mongoDB) CollectRawMetrics(location schema.Location) (*schema.Metric, e
 	}
 
 	// Processing confirmed case data
-	confirmedCount, confirmDiff, confirmDiffPercent, err := m.GetCDSConfirm(location)
+	confirmedCount, confirmDiff, confirmDiffPercent, err := m.GetCDSActive(location)
```

Comment:
Should this variable name be renamed to `activeCount`, `activeDiff` and `activeDiffPercent`, respectively.

Reply:
yes